docs/common/releases/release-notes.md

```diff
@@ -66,7 +66,21 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
 </dependencies>
 ```
 
-## 0.13.0 Release Notes
+## 0.13.1 Release Notes
+
+**Important bug fixes:**
+
+* **Spark**
+  * Fix predicate pushdown (filter-based and merge conditions) for row-level operations in Spark 3.2. [[\#4023](https://github.com/apache/iceberg/pull/4023)]
```

Comment:
we can follow the pattern used in 0.12.1, use sentences `#<PR Number> fixes something`

Reply:
Also prefer full sentence instead of using brackets

Reply:
Updated